ugvc/pipelines/mrd/prepare_data_from_mrd_pipeline.py

```diff
@@ -86,6 +86,12 @@ def parse_args(argv: list[str]) -> argparse.Namespace:
         default=None,
         help="""Basename of output files that will be created.""",
     )
+    parser.add_argument(
+        "--return_dataframes",
+        type=bool,
+        default=None,
+        help="""Whether to return dataframes instead of writing to file""",
+    )
```

Comment:
Why do we have this argument in the CLI? We never want the function to return a dataframe instead of writing when running from the CLI

Reply:
removed